packages/eslint-config-redux-saga/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@kilohealth/eslint-config-redux-saga",
+  "main": "./react-native.js",
```

Comment:
seems like we're missing some fields here

Reply:
True 😅